tests/unit_tests/db/test_db_handler.py

```diff
@@ -212,64 +212,76 @@ def test_copy_array_element_db(db, random_id, io_handler, model_version):
 def test_adding_new_parameter_db(db, random_id, io_handler, model_version):
     logger.info("----Testing adding a new parameter-----")
     test_model_version = "0.0.9876"
-    db.copy_array_element(
-        db_name=None,
-        element_to_copy="LSTN-01",
-        version_to_copy=model_version,
-        new_array_element_name="LSTN-test",
-        collection_name="telescopes",
-        db_to_copy_to=f"sandbox_{random_id}",
-        collection_to_copy_to="telescopes",
-    )
-    db.add_new_parameter(
-        db_name=f"sandbox_{random_id}",
-        array_element_name="LSTN-test",
-        version=test_model_version,
-        parameter="new_test_parameter_str",
-        value="hello",
-        collection_name="telescopes",
-    )
-    db.add_new_parameter(
-        db_name=f"sandbox_{random_id}",
-        array_element_name="LSTN-test",
-        version=test_model_version,
-        parameter="new_test_parameter_int",
-        value=999,
-        collection_name="telescopes",
-    )
+    tmp_par_dict = {
+        "parameter": None,
+        "instrument": "LSTN-test",
+        "site": "North",
+        "version": test_model_version,
+        "value": None,
+        "unit": None,
+        "type": None,
+        "applicable": True,
+        "file": False,
+    }
+
+    par_dict_int = copy.deepcopy(tmp_par_dict)
+    par_dict_int["parameter"] = "num_gains"
+    par_dict_int["value"] = 3
+    par_dict_int["type"] = "int64"
+    with pytest.raises(
+        ValueError,
+        match=re.escape("Value for column '0' out of range. ([3, 3], allowed_range: [1, 2])"),
```

Comment:
Why do we get here a list [3, 3]?

Reply:
The validation code converts everything into astropy table columns (even single numbers) to simplify the code. This allows to use the column range checks. So `[3,3]` is `{col_min}, {col_max}`. I think this should be ok.